src/main/java/xbot/common/command/BaseSetpointCommand.java

```diff
@@ -12,6 +12,8 @@ public BaseSetpointCommand(SupportsSetpointLock system, SupportsSetpointLock...
 
     @Override
     public boolean isFinished() {
-        return false;
+        // Setpoint commands end instantly by default since the set operation usually
+        // happens in initialize and then nothing else needs to happen
```

Comment:
Will this break any WhileTrue button presses? (i.e. something else will change the setpoint if this finishes?)